lib/ui/painting.dart

```diff
@@ -1568,6 +1568,17 @@ class Paint {
   }
 }
 
+/// The color space that an [Image] uses.
+enum ColorSpace {
+  /// The sRGB color gamut.
```

Comment:
Similarly here, if someone is looking at the documentation for `ColorSpace.sRGB`, I assume they already know that it means "the sRGB color gamut", so what's the question that led them to looking at the documentation? Maybe it's "what is sRGB", maybe it's "where do i specify this value"? Maybe it's "what range of colors are available for this"?

Reply:
I'm not sure what else to add, I said it's the standard colorspace of the web so that might help people understand.  It's kind of a complicated topic to insert in a docstring.  They'd be better off googling it beyond that.